client/state/sites/terms/reducer.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import merge from 'lodash/merge';
+import uniq from 'lodash/uniq';
+import get from 'lodash/get';
+import keyBy from 'lodash/keyBy';
+
+/**
+ * Internal dependencies
+ */
+import {
+	TERMS_RECEIVE,
+	DESERIALIZE
+} from 'state/action-types';
+import { isValidStateWithSchema } from 'state/utils';
+import {
+	termsSchema,
+	taxonomiesSchema
+} from './schema';
+
+export function taxonomies( state = {}, action ) {
+	switch ( action.type ) {
+		case TERMS_RECEIVE:
+			const currentTaxonomyTermIds = get( state, [ action.siteId, action.taxonomy ], [] );
+			// This feels a bit odd to me, but uncertain of best way to handle paginated data?
```

Comment:
>Orphaned term.IDs in localForage being one of my bigger concerns.

Not to suggest we rely on it too heavily, but our state persistence is busted after one week, so orphaned items won't remain forever present.